### ISSUE TITLE
multigrmpy.py: Illegal header name "depth sd" 

### ISSUE THREAD

**Issue by dantaki** [Illumina/paragraph#31]
Running a test of paragraph and I got this error

```
$ python3 paragraph/bin/multigrmpy.py -i test.vcf \
        -m HG03097.manifest.txt \
         -r GRCh38_full_analysis_set_plus_decoy_hla.fa \
         -o paragraph_test

Traceback (most recent call last):
  File "paragraph/bin/multigrmpy.py", line 353, in <module>
    main()
  File "paragraph/bin/multigrmpy.py", line 349, in main
    run(args)
  File "paragraph/bin/multigrmpy.py", line 249, in run
    raise Exception("Illegal header name %s. Allowed headers:\n%s" % (field, header_str))
Exception: Illegal header name depth sd. Allowed headers:
id,path,idxdepth,depth,read length,sex,depth variance
```

In the README, "sd depth" is one of the options, is this "idxdepth?" in the allowed headers? If I change "sd depth" to "idxdepth" the program begins to run. 

Thanks!

**Comment by traxexx** [comment]
idxdepth and "sd depth" are not the same thing.
"idxdepth" is the path to the output from running idxdepth, the depth estimation tool.
"depth sd" is the square root of "depth variance", used as a parameter in genotyping. If neither "depth variance" nor "depth sd" is specified, Paragraph will estimate that from your "depth" input.

I need to fix the error message and README.

**Comment by dantaki** [comment]
That's what I thought, and just to clarify I added those extra columns since I wanted Paragraph to know the sex of my samples since I am also using the X chromosome in genotyping. Therefore, I needed to put in those "depth sd" and "depth variance"

**Comment by traxexx** [comment]
Fix included in v2.4. now "depth sd" can be correctly recognized in manifest.
If you have "sex" column in the manifest, Paragraph will automatically adjust depth in chrX and chrY.
